tests/tools/tools/serialize.fs

```diff
@@ -52,6 +52,7 @@ module SerializationTests =
     let testExplicitLayoutArray() = 
         let mutable p1 = new Point()
         p1.Rho <- 1.0f
+        let p1 = p1
```

Comment:
To fix compilation on vs2013, mutable cannot be captured by closure.


Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone

-------- Original message --------
From: Weirong Zhu <notifications@github.com>
Date: 2015/12/08  19:06  (GMT-08:00)
To: MSRCCS/Prajna <Prajna@noreply.github.com>
Cc: sanjeevm0 <sanjeevm@stanfordalumni.org>
Subject: Re: [Prajna] Fix issue caused by early disposal of MemoryStreamB (#116)

> @@ -52,6 +52,7 @@ module SerializationTests =
>      let testExplicitLayoutArray() =
>          let mutable p1 = new Point()
>          p1.Rho <- 1.0f
> +        let p1 = p1

what's the purpose? to hide the mutable p1?

---
Reply to this email directly or view it on GitHub:
https://github.com/MSRCCS/Prajna/pull/116/files#r47046633